### ISSUE TITLE
Escape From Tarkov Crashing Loading In

### ISSUE THREAD

**Issue by AaronBottegal** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#564]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game / Application [Required]

Escape From Tarkov

### Game Platform [Required]

- [ ] Steam
- [ ] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [ ] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [X] Other (please specify)

### Other game platform

Battle State Games Launcher

### Processor / Processor Number [Required]

5800X3D

### Graphic Card [Required]

Arc A770 LE

### GPU Driver Version [Required]

- [X] 31.0.101.4887
- [X] 31.0.101.4885
- [ ] 31.0.101.4826
- [ ] 31.0.101.4824
- [ ] 31.0.101.4676
- [ ] 31.0.101.4672
- [ ] 31.0.101.4669
- [ ] 31.0.101.4644
- [ ] 31.0.101.4578
- [ ] 31.0.101.4577
- [ ] 31.0.101.4576
- [ ] 31.0.101.4575
- [ ] 31.0.101.4514
- [ ] 31.0.101.4502
- [ ] 31.0.101.4499
- [ ] 31.0.101.4382
- [ ] 31.0.101.4369
- [ ] 31.0.101.4335
- [ ] 31.0.101.4314
- [ ] 31.0.101.4311

### Rendering API [Required]

- [X] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build Number [Required]

- [ ] Windows 11 22H2
- [ ] Windows 11 21H2
- [X] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[SystemScan.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/13059743/SystemScan.txt)

### Description and steps to reproduce [Required]

Try to load into PMC Run on Customs, will crash. Offline loads in a lot more, but sometimes also crashes.

### Game graphic quality [Required]

- [ ] Low
- [ ] Medium
- [X] High
- [ ] Ultra

### Game resolution [Required]

1440p.

### Game VSync [Required]

On

### Game display mode [Required]

- [ ] Fullscreen
- [ ] Windowed
- [X] Borderless Windowed

### Detailed game settings [Required]

![2023-10-20 23-24  (0)](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/7392428/1b2f1402-1b33-4ed5-9c0d-8ff7a81e3c38)
![2023-10-20 23-24  (1)](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/7392428/a594b22d-8878-4c6f-ac55-b2827c9ae4d7)
![2023-10-20 23-24  (2)](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/7392428/e132b123-c6bd-420a-b5d2-06ef9e3c8abd)

### Device / Platform name

Windows 10

**Comment by Denizeri24** [comment]
also we need special driver optimizations for this game..

**Comment by AaronBottegal** [comment]
Will re-test on the latest driver that just came out Dec 7th and report back.

**Comment by Arturo-Intel** [comment]
@AaronBottegal hey, I was able to play with driver .5074 
can you please try with that driver?

**Comment by AaronBottegal** [comment]
Tried on the Dec 7th driver and current tarkov as of today, same everything, and it failed with a message which I believe I saw in Conan which was failing to allocate memory, and then it instantly blue screens with the error KERNEL_MODE_HEAP_CORRUPTION.

It was a scav on shoreline. I usually have most problems PMC'ing in on this, and I think it's the only second time, scav mode has crashed. But it all seems to be the same issues.

**Comment by Arturo-Intel** [comment]
@AaronBottegal 
I was able to play in [Practice Mode] https://youtu.be/VOfm9FeWnSE
but when I go for real the BSOD happens. 

-- r2

**Comment by AaronBottegal** [comment]
I get a blue screen/unity crash about 1/3rd of the load ins, usually takes a couple, even offline. Online thought it's like 75%+ crash rate.

Also a new tarkov game came out called Escape From Tarkov: Arena, and I can't get through a single round, lol. Loads in fine, crashes mid game most games.

**Comment by Arturo-Intel** [comment]
@AaronBottegal Hey, can you check if you have this behavior using the 101.5333 driver?
I just tried in the CUSTOMS map in an ONLINE session without any trouble.

**Comment by AaronBottegal** [comment]
I can load in maybe 10% of games, it triggers a huge memory leak in your driver. It's plain as day. I see 8GB used, 14-15reported, and at 16GB is just crashes the application. I'll try it in a couple hours here.

**Comment by AaronBottegal** [comment]
I tried to do a scav (Free timed run) on Streets and crashed to a blue screen before it got off the loading screen with Kernel Heap Mode Corruption, it's the same memory leak I always see.

Driver 31.0.101.5379 from 3 days ago.

**Comment by Denizeri24** [comment]
Yea, I get crash too..

**Comment by Denizeri24** [comment]
still crashing / 4K

**Comment by Arturo-Intel** [comment]
@AaronBottegal Did you try it with the latest driver? ([101.5989](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html)) 
Here in the lab we were able to play some rounds, no crashes

**Comment by AaronBottegal** [comment]
> @AaronBottegal Did you try it with the latest driver? ([101.5989](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html)) Here in the lab we were able to play some rounds, no crashes

Yeah, it seems the latest update and drivers don't crash. I don't think the driver got fixed, it was the bigger Tarkov update that just doesn't hit it now.

Sorta different issue, only thing I see now is corrupt textures (minor) and lasers in Tarkov Arena look like the sun 100% of the time/white screens. It seems changing the rendering resolution back and forth in it fixed the white screen issue, though. The lasers looking like the sun is constant, for yours and others.

**Comment by AaronBottegal** [comment]
Speak of the devil, just had a hard crash on Arena. No BSOD. Just monitors going out and the PC needed to be restarted 5 times to trigger BIOS safe mode to boot again, lol.

**Comment by AaronBottegal** [comment]
Yeah, Arena is now unplayable after today's update, just crashed in the first round again.

**Comment by rmorera-intel** [comment]
@AaronBottegal we have a new driver [32.0.101.6130](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html) could you please give it a try and report back?

**Comment by SavageWaffle** [comment]
Is this resolved, can we close?

**Comment by Felipe-Intel** [comment]
@Arturo-Intel please check. 
@SavageWaffle  please use phrases like "Are there updates about this issue?" so the support team or the OP can update accordingly.

**Comment by rmorera-intel** [comment]
Closing this one now, issue should be fixed since driver 32.0.101.6130.
If problem re-appears please open a new bug report.
